politeiawww/user/localdb/localdb.go

```diff
@@ -23,15 +23,14 @@ const (
 	sessionPrefix = "session:"
 )
 
-var (
-	_ user.Database = (*localdb)(nil)
-)
+var _ user.Database = (*localdb)(nil)
```

Comment:
These linter changes are driving me crazy.  Stop changing random stuff.

Reply:
Wasn't linter, added second var and then when deleted didn't revert to original, my bad, will revert to original formatting